pkg/apis/kops/instancegroup.go

```diff
@@ -196,6 +196,8 @@ type InstanceGroupSpec struct {
 	Packages []string `json:"packages,omitempty"`
 	// GuestAccelerators configures additional accelerators
 	GuestAccelerators []AcceleratorConfig `json:"guestAccelerators,omitempty"`
+	// MaxInstanceLifetime is the maximum amount of time, in seconds, that an instance can be in service.
+	MaxInstanceLifetime *int64 `json:"maxInstanceLifetime,omitempty"`
```

Comment:
maybe `*metav1.Duration` is a better type for this. so one can specify "1d30h" as value.

Reply:
I did the change b/c it really makes sense, though the authorized values are "ms", "s", "m", "h" not "d"